docs/QUICK_START.md

```diff
@@ -0,0 +1,201 @@
+- QGroundControl works well with Bluetooth on Android, Linux or Windows.  Create a new Bluetooth type connection via Application Settings -> Comm Links -> Add.
+- Mission Planner is recommended for Windows.  Connect to the COM port associated with the Bluetooth connection after pairing.
+
+UDP or TCP over WiFi is another popular wireless option.
+- All three of the recommended Tx modules could be connected to an external ESP32 board such as the M5 Stamp boards running the [Wireless Bridge](WIRELESS_BRIDGE.md) sketch.
+- All popular ground stations can connect via UDP.
+- The standard ports for MAVLink are 14550 for UDP and 5760 for TCP.
+
+All three of the recommended Tx modules include serial ports which may be used to connect to your ground station.
+- PC serial ports and RS232 serial USB adapters use higher voltage levels than the logic level required for serial connections on Tx modules.  You will need a TTL level USB serial adapter to connect your Tx serial port to a PC or mobile device.
+- The [MatekSys Tx modules](https://www.mateksys.com/?portfolio=mr24-30-tx#tab-id-2) provide 1 or 2 TTL level serial ports depending on the dip switch settings.
+- The FrSky R9M shares its single inverted serial port between CLI and MAVLink based on the position of [dip switch 1](FRSKY_R9.md#r9m-tx-module) at power on.  A [method](FRSKY_R9.md#diy-inverter-dongle) to invert the signals will be required.
+
+Direct USB connection is another useful option for both PCs and mobile devices which support OTG USB.
+- Unfortunately this option is not available with the FrSky R9M.
+- This option does not work out of the box with the recommended MatekSys Tx modules for this use case because the USB port on these modules is not accessible when installed in a radio JR bay.
+- It is possible to solder a type A USB cable directly to the USB pads on the back of the MatekSys Tx modules.
+- If you do this, you will need to flash the "siktelem" version of the firmware for your MatekSys Tx module.
+- If you solder a cable to these pads, you must then also use the soldered USB cable for flashing and not connect anything to the built-in USB-C jack.
+
+## 8. Radio calibration
+
+There are many calibration and setup steps required for ArduPilot on a new aircraft.  Here, we mention just the RC calibration step.  Not only because RC calibration needs to be repeated when switching to mLRS from a different RC system, but also because of a quirk you may encounter.
+
+- If you normally use QGroundControl as your ground station, be aware that it does not properly handle radio calibration when using "rc_override".
+- If you did not connect SBUS or CRSF RC output from your receiver to your flight controller, you should use another ground station such as Mission Planner or APM Planner 2.0 to perform radio calibration.
```

Comment:
I think these two bullets can be combined as the note about 'using rc_override' comes a bit out of nowhere to me.  

Perhaps:  

If you did not connect SBUS or CRSF RC output from your receiver to your flight controller and are using "rc_override" for your RC, you can't use QGroundControl to calibration your radio.  You will need to use another ground station such as Mission Planner or APM Planner 2.0 to perform radio calibration.

Reply:
I also changed the following bullet which talks about connecting the GCS via USB to replace "either" with "any" so I don't exclude QGC in the case where rc override is not used.